src/components/parameters/ParameterBasePath.svelte

```diff
@@ -59,7 +59,7 @@
         on:reset={() => dispatch('reset', formParameter)}
       />
     </Input>
-    <input type="file" on:change={onChange} use:useActions={use} />
+    <input {disabled} type="file" on:change={onChange} use:useActions={use} />
```

Comment:
Nice catch! Would it be better to just not show the file selection since it doesn't show what the current file is anyway?

Reply:
🫡 

<img width="277" alt="Screenshot 2024-09-04 at 2 11 04 PM" src="https://github.com/user-attachments/assets/96044656-105c-4956-a1e6-3a92742f0b4a">